src/main/webapp/WEB-INF/tags/instructor/moreInfo.tag

```diff
@@ -28,8 +28,8 @@
       <div class="panel-body">
         <span data-toggle="modal" data-target="#studentProfileMoreInfo"
             class="text-muted pull-right glyphicon glyphicon-resize-full cursor-pointer"></span>
-        <h5>More Info </h5>
         <%-- Note: When an element has class text-preserve-space, do not insert HTML spaces --%>
+        <h5><strong>More Info:</strong></h5>
```

Comment:
Couple of things:

- The comment at Line 31 is for the line 33, so better to keep them together and shift line 32 above the comment
- @damithc @whipermr5 Is the `:` after `More Info` necessary or should we remove it?

Reply:
> @damithc @whipermr5 Is the : after More Info necessary or should we remove it?

Can keep.

Reply:
No worries! I shifted the commented line down to 32.